### DISCUSSION TITLE
"I thought it was obvious to you," he said. "No." | /en/2017/10/not-obvious/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#966] [Comments]
https://yihui.org/en/2017/10/not-obvious/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
I'd rather people treat me as a 5-year old when they report issues :)

> Originally posted on 2017-10-15 03:42:32

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
You can use the menu "Edit this page" on the left to help me fix anything on pretty much any of my web pages: <https://yihui.name/en/2013/06/fix-typo-in-documentation/>

> Originally posted on 2017-10-17 02:04:25

**Reply by yihui** [reply]
No, no need to apologize. Thanks for the help!

> Originally posted on 2017-10-18 14:46:11

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
That is a very thoughtful blog post. I learned a lot from it. Thanks for sharing!

> Originally posted on 2017-10-18 14:45:47

[a comment by an automated account is not shown]

**Comment by Vechus** [comment]
why was I mentioned in this discussion?

**Reply by yihui** [reply]
Yesterday several Github accounts were spammed suddenly and the spammer mentioned you randomly. Github has taken action to delete the spams, which is why you got the Github notification but don't see anything related to you here. Sorry about that.
